website/docs/components/copy/snippet/partials/guidelines/overview.md

```diff
@@ -0,0 +1 @@
+Use `Copy::Snippets` when it speeds up or imrpoves the user's workflow. Remember that users can still copy conetent manually, so take care not to overload the UI with them. 
```

Comment:
@andgen404 and I discussed this one yesterday. These 2 lines are meant to replace the whole "Effective Usage" section he had initially, where the main goal was to indicate exactly when a consumer should use a copy snippet and guide away from overuse of the component. Essentially we want them to use the Copy Snippet intentionally and when it helps the end-user complete a necessary job. I don't think your version gets to that goal because it takes out the "warning" part. 

I'd also argue that the Copy Snippet shouldn't be used for plain text strings as there's no longer a variant for this.